### PR TITLE
#116 - correct urlpatterns for offer

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

File: volontulo/urls.py [hunk 1]
```diff
@@ -23,27 +23,37 @@
...
     url(
-        r'^offers/form/(?P<organization_id>[0-9]+)$',
+        r'^offers/create$',
         views.offer_form,
         name='offer_form'
     ),
+    # it should be moved to offers/<slug>/<id>/edit:
     url(
-        r'^offers/apply/(?P<offer_id>[0-9]+)$',
+        r'^offers/activate/(?P<offer_id>[0-9]+)$',
```
Comment (Yog) [inline volontulo/urls.py:39]: shouldn't this url follow `/offers/<slug>-<ID>/join` pattern?

Comment (magul) [inline volontulo/urls.py:39]: Activate should be handled by edit. We don't have right now this functionality in edit, so I left it here untouched.

Comment (filipgorczynski) [inline volontulo/urls.py:39]: Apply should be there because volunteers can apply for offers. It's different action than activating.

Comment (magul) [inline volontulo/urls.py:39]: But instead of applu we use verb join. They the same thing in our functionality.

**Pull request merged**